package.json

```diff
@@ -99,6 +99,7 @@
     "babel-jest": "^24.9.0",
     "core-js": "^3.5.0",
     "coveralls": "^3.0.9",
+    "eslint-plugin-react": "^7.17.0",
```

Comment:
I am not sure what I did, but I needed to **reinstall** this package while working on this PR